src/pages/TransactionDetails.vue

```diff
@@ -44,24 +42,24 @@ const transactionHash = computed(() => {
   return hash;
 });
 
-const transaction = ref<DetailedTransaction | null>(null);
-const isFetchingTransaction = ref(false);
-
-watch(
-  () => transactionHash.value,
-  async () => {
-    try {
-      isFetchingTransaction.value = true;
-
-      transaction.value = await http.fetchTransaction(transactionHash.value);
-    } catch (e) {
-      handleUnknownError(e);
-    } finally {
-      isFetchingTransaction.value = false;
-    }
+const transactionScope = useParamScope(
+  () => {
+    return {
+      key: transactionHash.value,
+      payload: transactionHash.value,
+    };
   },
```

Comment:
Primitive payloads could be passed as-is (they themselves will be used as the key): ([docs](https://www.npmjs.com/package/@vue-kakuyaku/core#reactively-parametrised-scope-with-useparamscope)):

```suggestion
  transactionHash,
```